to_copy/usr/bin/pep-filter.sh

```diff
@@ -0,0 +1,268 @@
+            echo "${ARRAY[*]}" | ipset restore
+        IFS="$OLDIFS"
+
+        /sbin/iptables-save | grep -Eq -- "--match-set \<$list\>" || {     
+            /usr/sbin/iptables -I ENCRYPTME 2 -m set --match-set "$list" dst -j DROP \
+            || fail "Failed to insert iptables rule $list"
+        }
+    done
+}
+
+
+add_domains() {
+    local list_name="$1"
+    local new_domain_file="$2"
+    local tmp_domain_file="$TMP_DIR/domains.old"
+    local domain_file="$FILTERS_DIR/$list_name.blacklist"
+
+    touch "$tmp_domain_file" || fail "Failed to create temp domain file"
+    mkdir -p "$FILTERS_DIR" \
+        || fail "Failed to create blacklists directory"
+
+    # keep things clean add keep dupes scrubbed out as we update the domain list
+    [ -s "$domain_file" ] && \
+        cat "$domain_file" | sort -u > "$tmp_domain_file"
+
+    cat "$tmp_domain_file" "$new_domain_file" | sort -u > "$domain_file" \
+        || fail "Failed to write $domain_file"
+    reload_filter \
+       || fail "Failed to reload dns-filter"
+}
+
+
+prune_list() {
+    local list_name="$1"
+    local domain_file="$FILTERS_DIR/$list_name.blacklist"
+
+    # delete the IP table rule and ipset list
+    /sbin/ipset -n list | grep -E "$list_name\.[0-9]{2}" | while read sublist; do
+        /sbin/iptables-save | grep -Eq -- "--match-set \<$sublist\>" && {     
+            /sbin/iptables -D ENCRYPTME -m set --match-set "$sublist" dst -j DROP \
+               || fail "Failed to delete iptables rule for the list $sublist"
+        }
+        /sbin/ipset destroy "$sublist" \
+           || fail "Failed to delete ipset $sublist"
+    done
+
+    # delete a domain blacklist file
+    [ -f "$domain_file" ] && {
+       rm -f "$domain_file"
+       reload_filter
+    }
+    return 0
+}
+
+
+reset_filters() {
+    # delete all ipset lists and iptables rules
+    /sbin/ipset -n list | while read list_name; do
+        /sbin/iptables-save | grep -Eq -- "--match-set \<$list_name\>" && {     
+            /sbin/iptables -D ENCRYPTME -m set --match-set "$list_name" dst -j DROP \
+               || fail "Failed to delete iptables rule for the list $list_name"
+        }
+        /sbin/ipset destroy "$list_name" \
+           || fail "Failed to delete ipset $list_name"
+    done
+
+    # remove our domain blacklists
+    rm -rf "$FILTERS_DIR" \
+        || fail "Failed to delete domain lists"
+    reload_filter
+}
+
+
+# reads stdin to parse IPv4 CIDR ranges and domain names and filter them out
+append_list() {
+    local list_name="$1"
+    local cidr_file="$TMP_DIR/$list_name.cidr"
+    local domain_file="$TMP_DIR/$list_name.domains"
+    local stdin="$TMP_DIR/$list_name.stdin"
+
+    cat > "$stdin"
+    cat "$stdin" | grep -E "$CIDR_RE" > "$cidr_file"
+    cat "$stdin" | grep -E "$DOMAIN_RE" > "$domain_file"
+
+    [ -s "$cidr_file" ] &&  add_ips "$list_name" "$cidr_file"
+    [ -s "$domain_file" ] &&  add_domains "$list_name" "$domain_file"
+}
+
+
+unbound_health() {
```

Comment:
This can be simplified to simply `pgrep unbound &>/dev/null`, which doesn't even need a function; same with `filter_server_health`

Reply:
unbound_health and pep_filter_health functions were removed from pep-filter. This logic is done in peps-docker-stats